### ISSUE TITLE
It should be possible to enter (arbitrarily shifted) logprobs and/or odds ratos instead of just probabilities

### ISSUE THREAD

**Issue by JasonGross** [Sage-Future/fatebook#71]

**Comment by jonnyspicer** [comment]
I agree this would be neat, sadly I don't think it'll be near the top of our priority list any time soon. If you'd like to submit a PR yourself, at least the odds ratios ought to be fairly easy to implement - I would look at `components/questions/UpdateableLatestForecast.tsx` and `components/predict-form/PredictionPercentageInput.tsx`
